### ISSUE TITLE
mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError

### ISSUE THREAD

**Issue by dinhkk** [nginxinc/nginx-amplify-agent#101]
I am using Mariadb version 10.3.5
After activate the mysql plugins. It is displaying error message in log 
/var/log/amplify-agent/agent.log
```
2018-04-11 03:41:43,817 [28949] mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError
2018-04-11 03:41:53,823 [28949] mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError
2018-04-11 03:42:03,832 [28949] mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError
2018-04-11 03:42:13,844 [28949] mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError
2018-04-11 03:42:23,847 [28949] mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError
2018-04-11 03:42:33,858 [28949] mysql_metrics mysql_metrics failed to collect: mysql_status raised OperationalError
```

```
2018-04-11 03:57:29,274 [2020] mysql_metrics mysql_metrics failed to collect: mysql_status raised TypeError
2018-04-11 03:57:39,277 [2020] mysql_metrics mysql_metrics failed to collect: mysql_status raised TypeError
2018-04-11 03:57:49,280 [2020] mysql_metrics mysql_metrics failed to collect: mysql_status raised TypeError
2018-04-11 03:57:59,297 [2020] mysql_metrics mysql_metrics failed to collect: mysql_status raised TypeError
2018-04-11 03:58:09,305 [2020] mysql_metrics mysql_metrics failed to collect: mysql_status raised TypeError
```

And the dashboard can not show the mysql metrics

**Comment by akamoroz** [comment]
Hi @dinhkk ,
this issue requires some investigation. Creating a ticket in our support queue. Pls check your mailbox.

**Comment by dinhkk** [comment]
Hi @akamoroz,
Thank you so much for your support.
I investigated the issue.
That's caused by the difference of setting for connecting to MariaDB socket. It's in the configuration file : /etc/amplify-agent/agent.conf
The default setting :  unix_socket = /var/run/mysqld/mysqld.sock
But my server environment : unix_socket = /var/lib/mysql/mysql.sock

But I saw that , if I try to connect via port 3306, and comment out the unix socket. It's still showing error. 
```
host = localhost
port = 3306
```

So this issue was fixed by changing unix socket to : /var/lib/mysql/mysql.sock

thank you
